apply/templates/apply/accessibility_statement.html

```diff
@@ -30,7 +30,7 @@ <h2 class="govuk-heading-m">{% trans %}Preparation of this accessibility stateme
                 {% trans %}additional manual testing and our responses to the findings of these activities.{% endtrans %}</p>
             <h2 class="govuk-heading-m">{% trans %}Feedback and contact information{% endtrans %}</h2>
             <p class="govuk-body">{% trans %}If you find any problems that are not listed on this page or you think we’re not meeting the accessibility requirements,{% endtrans %}
-                <a href="mailto:fundingservice.support@communities.gov.uk" class="govuk-link">{% trans %}contact us{% endtrans %}</a>.</p>
+                <a href="https://mhclgdigital.atlassian.net/servicedesk/customer/portal/5/group/68" class="govuk-link">{% trans %}contact us{% endtrans %}</a>.</p>
```

Comment:
Can we move this URL into Config as we are hardcoding it in several locations?

Reply:
Good shout, created a PR for this: https://github.com/communitiesuk/funding-service-pre-award-frontend/pull/38